src/radical/pilot/agent/lm/__init__.py

```diff
@@ -2,7 +2,7 @@
 __copyright__ = "Copyright 2016, http://radical.rutgers.edu"
 __license__   = "MIT"
 
-
-from .base import *
-from .base import LaunchMethod
-
+# pylint: disable=wildcard-import
+from .base import *  # noqa: F403, F401
+from .base import LaunchMethod  # noqa: F403, F401
+# pylint: enable=wildcard-import
```

Comment:
Yes, we can do something like that. I am not sure whether we can say that this is priority or not. If we have a continuously failing Travis, because this is not fixed, then I am not sure how automating testing and everything with code quality helps. Then how are you going to enforce it for new pushes?

Nonetheless, it should be a priority for version 2

Reply:
So, I may be missing the point, but I don't think the idea is to configure the tests so that they pass.  If they show problems, then that's *good*, and they should continue to do so until they are fixed.  If we only care about green boxes, then `assert(True)` will do the trick ;-)

So I'd rather use a pylintrc which reflects what we want in terms of code checks, instead of one which is green and disabled in some code only so that the boxes turn green.  What's the point?

I am surprised that this is me saying this, who like tests the least amongst all of us ;-)

Reply:
I agree. `from ..some.module.name import *` is commonly deprecated and we should avoid it.